integration/pom.xml

```diff
@@ -46,6 +46,19 @@
                 <module>broker-tests</module>
             </modules>
         </profile>
+        <profile>
+            <id>smoke-tests</id>
+            <properties>
+                <testng.suiteXml.location>src/test/resources/smokeTestng.xml</testng.suiteXml.location>
+            </properties>
+            <modules>
+                <module>automation-extensions</module>
+                <module>mediation-tests</module>
+                <module>business-process-tests</module>
+                <module>dataservice-hosting-tests</module>
+                <module>broker-tests</module>
+            </modules>
```

Comment:
Do we need to have the modules section inside smoke-tests profile since it is common for all profiles?

Reply:
It is not common to all profiles. E.g. mediation-profile, broker-profile

Reply:
:+1: